ansible/roles_ocp_workloads/ocp4_workload_lpe_3scale/tasks/remove_workload.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: virtualenv path to use for pip
+  set_fact:
+    pip_path: "{{ ansible_python_interpreter | dirname }}"
+
+- name: "Uninstall lxml module from python"
```

Comment:
I don't think you have to worry about that one. You may have more than one instance of this workload running on the cluster.